cvmfs/receiver/payload_processor.cc

```diff
@@ -67,11 +67,15 @@ void PayloadProcessor::ConsumerEventCallback(
     path = event.object_name;
   } else {
     // kEmpty - this is an error.
-    LogCvmfs(kLogCvmfs, kLogStderr, "Event received with unknown object.");
+    LogCvmfs(kLogReceiver, kLogDebug | kLogSyslogErr,
+             "Event received with unknown object.");
     num_errors_++;
     return;
   }
 
+  LogCvmfs(kLogReceiver, kLogDebug | kLogSyslog,
```

Comment:
Would it make sense to put this only in the debug log?  I can imagine that under load it quickly fills syslog.  We can also attach a TODO here if we want to do it later.